FHEM/14_LTECH.pm

```diff
@@ -324,7 +324,7 @@ sub Set($@)
             my $rgbcolor_sel = Color::hsv2hex($h,$s,$v);
             readingsSingleUpdate($hash, 'rgbcolor_sel', $rgbcolor_sel, 1);
             $args[0] = "on"; 
-            Set($hash, $name, @args);
+            &Set($hash, $name, @args);
```

Comment:
Die Prototypen befinden sich in der Definition, nicht im Aufruf 
Also eher

`sub set($$$) {` durch `sub set {` ersetzen und dann die übergebenen Parameter validieren

z.B.

```
my $hash = shift;
return if ref($hash) ne 'HASH' ;
my $name = shift // return;
...

```